src/serviceregistry/manual/examples.md

```diff
@@ -139,8 +136,26 @@ This file contains specific examples of what payloads are expected and returned
 
```

Comment:
In revoke device, that is not query parameter, but path parameter. (For some reason, I can't comment on that particular line.)